Cargo.toml

```
@@ -37,7 +37,7 @@ async-trait = "0.1.50"
 bs58 = "0.4.0"
 cfg-if = "1.0.0"
 data-encoding = "2.3.2"
-didcomm-rs = { git = "https://github.com/evannetwork/didcomm-rs.git", default-features = false }
+didcomm-rs = { git = "https://github.com/evannetwork/didcomm-rs.git", cargdefault-features = false }
```

This looks like a new syntax. :D

```suggestion
didcomm-rs = { git = "https://github.com/evannetwork/didcomm-rs.git", default-features = false }
```